src/App.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import {
+  BrowserRouter as Router,
+  Route
+} from 'react-router-dom'
```

Comment:
avoid unnecessary imports here, you only ever use `BrowserRouter` in index.js to wrap App, so it's not necessary within App.js